package/MDAnalysis/visualization/streamlines.py

```diff
@@ -170,72 +199,91 @@ def produce_list_centroids_this_frame(list_indices_in_polygon):
+    dy_array : array of floats
             An array object containing the displacements in the y direction
-        **average_displacement** : float
-            :math:`\\frac {\\sum \\sqrt[]{dx^2 + dy^2}} {N}`
-        **standard_deviation_of_displacement** : float
-            standard deviation of :math:`\\sqrt[]{dx^2 + dy^2}`
+    average_displacement : float
+            :math:`\frac{\sum\sqrt[]{dx^2 + dy^2}}{N}`
+    standard_deviation_of_displacement : float
+            standard deviation of :math:`\sqrt[]{dx^2 + dy^2}`
 
-    :Examples:
-
-    ::
+    Examples
+    --------
+    Generate 2D streamlines and plot::
 
         import matplotlib, matplotlib.pyplot, np
         import MDAnalysis, MDAnalysis.visualization.streamlines
-        u1, v1, average_displacement,standard_deviation_of_displacement =
-        MDAnalysis.visualization.streamlines.generate_streamlines('testing.gro','testing_filtered.xtc',grid_spacing =
-        20, MDA_selection = 'name PO4',start_frame=2,end_frame=3,xmin=-8.73000049591,xmax= 1225.96008301,
-        ymin= -12.5799999237, ymax=1224.34008789,maximum_delta_magnitude = 1.0,num_cores=16)
-        x = np.linspace(0,1200,61)
-        y = np.linspace(0,1200,61)
+
+        u1, v1, average_displacement, standard_deviation_of_displacement =
+            MDAnalysis.visualization.streamlines.generate_streamlines('testing.gro', 'testing_filtered.xtc',
+                    grid_spacing=20, MDA_selection='name PO4', start_frame=2, end_frame=3,
+                    xmin=-8.73000049591, xmax= 1225.96008301,
+                    ymin= -12.5799999237, ymax=1224.34008789,
+                    maximum_delta_magnitude=1.0, num_cores=16)
+        x = np.linspace(0, 1200, 61)
+        y = np.linspace(0, 1200, 61)
         speed = np.sqrt(u1*u1 + v1*v1)
         fig = matplotlib.pyplot.figure()
-        ax = fig.add_subplot(111,aspect='equal')
+        ax = fig.add_subplot(111, aspect='equal')
         ax.set_xlabel('x ($\AA$)')
         ax.set_ylabel('y ($\AA$)')
-        ax.streamplot(x,y,u1,v1,density=(10,10),color=speed,linewidth=3*speed/speed.max())
+        ax.streamplot(x, y, u1, v1, density=(10,10), color=speed, linewidth=3*speed/speed.max())
         fig.savefig('testing_streamline.png',dpi=300)
 
     .. image:: testing_streamline.png
 
-    .. [Chavent2014] Chavent, M.\*, Reddy, T.\*, Dahl, C.E., Goose, J., Jobard, B., and Sansom, M.S.P. (2014)
-       Methodologies for the analysis of instantaneous lipid diffusion in MD simulations of large membrane systems.
-       *Faraday Discussions* **169**: **Accepted**
+
+    References
+    ----------
+    .. [Chavent2014] Chavent, M.*, Reddy, T.*, Dahl, C.E., Goose, J., Jobard,
+       B., and Sansom, M.S.P. Methodologies for the analysis of instantaneous
+       lipid diffusion in MD simulations of large membrane systems.  *Faraday
+       Discussions* **169** (2014), 455–475. doi: `10.1039/c3fd00145h`_
+
+    .. _`10.1039/c3fd00145h`: https://doi.org/10.1039/c3fd00145h
```

Comment:
That's the DOI. I have always been putting hrefs on the dois themselves because in this way the important semantic unit is highlighted (as opposed to "click here"-type links). See, for example, http://www.mdanalysis.org/mdanalysis/documentation_pages/references.html#citations-for-the-whole-mdanalysis-library